docs/stream-aggregation.md

```diff
@@ -308,6 +308,48 @@ The resulting histogram buckets can be queried with [MetricsQL](https://docs.vic
 See [the list of aggregate output](#aggregation-outputs), which can be specified at `output` field.
 See also [quantiles over input metrics](#quantiles-over-input-metrics) and [aggregating by labels](#aggregating-by-labels).
 
+### Aggregating histograms
```

Comment:
I believe that it shoud be added to the list in "Use cases" section.

Reply:
updated